src/sp_wrapper.c

```diff
@@ -0,0 +1,31 @@
+#include "php_snuffleupagus.h"
+#include "sp_config.h"
+
+ZEND_DECLARE_MODULE_GLOBALS(snuffleupagus)
+
+void sp_disable_wrapper() {
+  HashTable *orig = php_stream_get_url_stream_wrappers_hash();
+  HashTable *full = pemalloc(sizeof(*full), 1);
```

Comment:
Why `full`?

Reply:
It is the full hashtable, with every entry.

Reply:
`orig_hashtable_complete`?